package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "lint": "eslint '**/*.js'",
     "lint:fix": "prettier-eslint '**/*.js' --write",
-    "test:staged": "nyc mocha --timeout 1000 -r esm --exit && nyc check-coverage --statements 100 --functions 100 --lines 100 --branches 100",
+    "test:staged": "npm test && nyc check-coverage --statements 100 --functions 100 --lines 100 --branches 100",
```

Comment:
Why do we have this change?

Reply:
I changed it because it is the same as the test script